src/UITests/GettingStartedPage.cs

```diff
@@ -9,29 +9,33 @@ public class GettingStartedPage : AIWinSession
     {
         [TestMethod]
         [TestCategory("NoStrongName")]
-        public void VerifyGettingStartedTitle()
+        public void GettingStartedPageTests()
+        {
+            VerifyGettingStartedTitle();
+            VerifyAccessibility();
+        }
+
+        private void VerifyGettingStartedTitle()
         {
             Assert.AreEqual("Accessibility Insights for Windows - ", driver.Title);
         }
 
-        [TestMethod]
-        [TestCategory("NoStrongName")]
-        public void VerifyAccessibility()
+        private void VerifyAccessibility()
         {
-            var result = driver.ScanAIWin();
+            var result = driver.ScanAIWin(TestContext.ResultsDirectory);
```

Comment:
Out of curiosity, do we know if this is sufficient to percolate the a11ytest file to ADO if there's a failure?

Reply:
I do not know.